src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form:hover {
```

Comment:
Do you really need the hover state? I think you need just focus on inputs.

Reply:
Yes, hover is required.
Because the task mentioned three states, including hover.
Should I delete hover?